IdokladSdk/Models/Common/NullableProperty.cs

```diff
@@ -107,5 +109,56 @@ public override int GetHashCode()
             hash = (hash * 23) + Value.GetHashCode();
             return hash;
         }
+
+        /// <inheritdoc />
+        public TypeCode GetTypeCode() => OriginalValue.GetTypeCode();
```

Comment:
Nechtěl bys tu strukturu udělat partial a tyto metody, kterými implementuješ to rozhraní IConvertible hodit do druhého souboru?